### DISCUSSION TITLE
Support for dynamic timeScale via TimeManager (respected by CSP)

### DISCUSSION THREAD

**Discussion by lordubbe** [FirstGearGames/FishNet#225] [Feature Request]
Many games need dynamic timeScale and currently there is no built-in support for it, and no immediately obvious way to make it work especially using the CSP system. There's already a TimeManager that would be a logical place to interface with for this!

**Comment by FirstGearGames** [comment]
Per our discussion in the server this will not be added. There are way too many complications. I still recommend coding this specifically to your game.
